### ISSUE TITLE
api of the differents ranking

### ISSUE THREAD

**Issue by Dalethvien** [thewca/worldcubeassociation.org#6701]
I think it could be nice to have an api with the ranking of the different event, if that's possible, like with the top 100 average/single of each event.

**Comment by gregorbg** [comment]
I agree that it would be cool, but unfortunately such a convenience feature is very low on our priority list. Thanks for the suggestion!
